.github/workflows/send-transactions-test.yml

```diff
@@ -0,0 +1,586 @@
+name: Test commit
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - develop
+      - hotfix/*
+      - release/*
+
+jobs:
+  build:
+    name: Test
+    runs-on: ubuntu-20.04
+    timeout-minutes: 45
+    strategy:
+      matrix:
+        java:
+          - zulu@1.8.282
+          - openjdk@1.11.0
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.14.2
+          cache: 'npm'
+          cache-dependency-path: .github/action_scripts/package-lock.json
+      - name: Setup Java and scala
+        uses: olafurpg/setup-scala@v13
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Cache for sbt & coursier ♨️
+        uses: coursier/cache-action@v6
+        id: coursier-cache
+
+      - name: Generate JARS
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          sbt core/assembly keytool/assembly wallet/assembly dagL1/assembly currencyL0/assembly currencyL1/assembly
+          
+          mv modules/core/target/scala-2.13/tessellation-core-assembly-* .github/config/containers/jars
+          mv modules/keytool/target/scala-2.13/tessellation-keytool-assembly-* .github/config/containers/jars
+          mv modules/wallet/target/scala-2.13/tessellation-wallet-assembly-* .github/config/containers/jars
+          mv modules/dag-l1/target/scala-2.13/tessellation-dag-l1-assembly-* .github/config/containers/jars
+          mv modules/currency-l0/target/scala-2.13/tessellation-currency-l0-assembly-* .github/config/containers/jars
+          mv modules/currency-l1/target/scala-2.13/tessellation-currency-l1-assembly-* .github/config/containers/jars
+          
+          cd .github/config/containers/jars
+          
+          mv tessellation-core-assembly-* global-l0.jar
+          mv tessellation-keytool-assembly-* cl-keytool.jar
+          mv tessellation-wallet-assembly-* cl-wallet.jar
+          mv tessellation-dag-l1-assembly-* dag-l1.jar
+          mv tessellation-currency-l0-assembly-* currency-l0.jar
+          mv tessellation-currency-l1-assembly-* currency-l1.jar
+
+
+      - name: Running global-l0
+        env:
+          CL_KEYSTORE: token-key.p12
+          CL_KEYALIAS: token-key
+          CL_PASSWORD: ${{ secrets.P12_GENESIS_FILE_PASSWORD }}
+        run: |          
+          cd .github/config/containers/global-l0
+          java -jar ../jars/cl-keytool.jar generate
+          
+          export CL_PUBLIC_HTTP_PORT=9000
+          export CL_P2P_HTTP_PORT=9001
+          export CL_CLI_HTTP_PORT=9002
+          export CL_APP_ENV=dev
+          export CL_ENV=dev
+          export CL_COLLATERAL=0
```

Comment:
I see you pass some attributes as cmd arguments and some as exported env variables. I think you can unify it either to env variables or cmd line 🤔 just food for thought